### PR TITLE
New Issue - Disconnect Between Business and IT

### PR DESCRIPTION
**Pull request by Kristynaslomkova**

### CODE REVIEW

**Comment by Kristynaslomkova** [comment]
@prokopsimek Can you please add Aspect and Stage? I am not sure what to fill. Thanks!

File: issues/disconnect-between-business-and-it.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- business
+- innovation
+- finance
+- Kovarus
+- investment
+- budget
+---
+# Disconnect Between Business and IT
+
+## What Is Disconnect Between Business and IT
```
Comment (prokopsimek) [inline issues/disconnect-between-business-and-it.md]: I am not sure about headlines. Is that correct `What Is Disconnect Between Business and IT`? Shouldn't it be `What Is Disconnection Between Business and IT` or `What Means Disconnection Between Business and IT`?

Comment (Kristynaslomkova) [reply issues/disconnect-between-business-and-it.md]: I think I just missed the article. But "disconnect between business and IT" is a phrase, if I'm correct.

Comment (prokopsimek) [reply issues/disconnect-between-business-and-it.md]: Ok, thanks for the explanation. :)

**Pull request merged**